src/js/components/DataTable/Resizer.js

```diff
@@ -81,27 +87,46 @@ const Resizer = ({ onResize, property }) => {
     };
   }
 
+  const onKeyDown = useCallback(
+    (event) => {
+      event.preventDefault();
+      if (!ref.current) return;
+      let element = ref.current;
+      while (element && element.nodeName !== 'TH') element = element.parentNode;
+      const currentWidth = element.getBoundingClientRect().width;
+      const propertyWidth = widths?.[property];
+      // Used 12 here to align with the value set in onMouseMove
+      const delta = event.key === 'ArrowLeft' ? -12 : 12;
+      onResize(property, (propertyWidth || currentWidth) + delta);
```

Comment:
One thing I'm noticing is if I use the left arrow to reduce the width of a column and the column reaches a min-width, then when I hit right arrow it doesn't immediately increase the width of the column.

Let's say I get to the min-width of a column (not necessarily a defined min-width but just what there's space for), and I hit left arrow 5 times. It will take me 6 presses of right arrow for the column to increase in width again.

Seems odd given what I see in the logic but maybe the property width is coming back smaller than it should be?

Reply:
Good catch, just pushed up a change that should resolve this